docs/spelling_wordlist.txt

```diff
@@ -36,6 +36,7 @@ iterable
 Koblitz
 Lange
 metadata
+multi
```

Comment:
In docker infra the "multi-threaded" line in the changelog threw a spelling error.